modules/ROOT/pages/configure-service-mesh.adoc

```diff
@@ -0,0 +1,143 @@
+= Configure Service Mesh
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: service mesh, microservices, downloading, installing
+
+image::configure-service-mesh-breadcrumb.png[]
+
+The following topics are covered in this section:
+
+* Provision Service Mesh Adapter
+* Bind Service Mesh Adapter
+
+== Service Mesh Adapter Overview
+
+After you install Service Mesh, you must provision the Service Mesh adapter and bind it with a specific namespace. When you provision the adapter, you must specify the size of the adapter based on the number of CPUs and memory of the system that will be managed in Service Mesh.
```

Comment:
Please mention. clearly here - that adapters are namespace bound, each namespace needs a separate adapter

Reply:
Done!